test_util/terraform_test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+if [ $# -lt 1 ]; then
+    echo "Usage: $0 <test group>"
+    exit 1
+fi
+
+set +e
+set -x
+set -o pipefail
+
+MASTER_IP=$(./terraform output --json masters-ips |jq -r '.value[0]')
+ssh_user=$(./terraform output --json -module dcos.dcos-infrastructure masters.os_user |jq -r '.value')
+ssh_user=${ssh_user:-$(./terraform output --json -module dcos.dcos-infrastructure masters.admin_username |jq -r '.value')}
+
+SSH="ssh  -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ./id_rsa ${ssh_user}@$MASTER_IP"
+PYTEST='source /opt/mesosphere/environment.export && cd `find /opt/mesosphere/active/ -name dcos-integration-test* | sort | tail -n 1` && '
+
+function run_pytest {
+     SSH_ARGS="$PYTEST $@"
+     $SSH -R 9411:127.0.0.1:9411 -- $SSH_ARGS
```

Comment:
Why are we doing this port forwarding? A comment here will be helpful.

Reply:
Ah, it's from an experiment, I'll drop it.